tests/integration/pilot/testdata/upgrade/1.6.11-install.yaml

```diff
@@ -0,0 +1,1264 @@
+# GENERATED by running "istioctl manifest generate -f minimal.yaml --set revision=1-6-11" on version 1.6.11 istioctl where "minimal.yaml" is:
+# apiVersion: install.istio.io/v1alpha1
+# kind: IstioOperator
+# spec:
+#   components:
```

Comment:
nit: can you add `hub: gcr.io/istio-release`? To avoid dockerhub rate limit + faster pulls since we run in GKE